src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -352,6 +352,12 @@ Plugin::Plugin()
           [&](const Config& config) {
               return _metrics->GetGops(get_specified_device_name(config));
           }}},
+        {ov::execution_devices.name(),
+         {true,
+          ov::PropertyMutability::RO,
+          [&](const Config& config) {
+              return std::string("NPU");
```

Comment:
if you have multiple device, it should be concrete device with so-called `device_id`

Reply:
added device_id to the string. thanks!